bookShop/src/app/book-component/book-component.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="book-wrapper" *ngFor="let item of bookData">
```

Comment:
It's better to name `book` variable instead of `item`

Reply:
Corrected

Reply:
corrected